### ISSUE TITLE
Error with rx

### ISSUE THREAD

**Issue by scmbradley** [emacs-origami/origami.el#22]
I installed the package from CELPA, and this is the error I got: 

`origami-parsers.el:782:1:Error: Unknown rx form ‘beg-tag’`

I experimented briefly and I can fold python files at top-level nodes, but not at lower level nodes (so the behaviour is the same as the original `origami` in that respect). 

Here's the `list-packages` version information:

```
     Status: Installed in ‘origami-20210322.1930/’ (unsigned). Delete
    Version: 20210322.1930
     Commit: 921d15cf398877f67b2b91e4eadb61d358037c40
    Summary: Flexible text folding
   Requires: emacs-24.4, s-1.9.0, dash-2.5.0
   Homepage: https://github.com/emacs-origami/origami.el
   Keywords: folding 
Other versions: 20210322.1930 (celpa), 20200331.1019 (melpa).

```
Here's the output of `M-x version`: `GNU Emacs 26.3 (build 2, x86_64-pc-linux-gnu, GTK+ Version 3.24.14) of 2020-03-26, modified by Debian`

**Comment by jcs090218** [comment]
Hi, can you post the python code that causes this issue, so we can try to reproduce it. Thanks! :)

**Comment by scmbradley** [comment]
That error appears when I install the package through CELPA.(so not related to any particular file) 

Perhaps the non functioning python folding is a separate issue?

**Comment by jcs090218** [comment]
Ah... I can reproduce this issue. The reason why this is reporting the error is due to the incompatibility of `rx.el` from Emacs built-in lisp module. I currently don't have a solution yet but the workaround is to use a newer Emacs version (27.1 or above). If I have found the solution then I will apply a patch to support version Emacs 26.x ASAP.

Sorry for the inconvenience!

**Comment by scmbradley** [comment]
That must be it, yes. I'm using emacs available through the main Debian repo, which is `26.3` currently.
